modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9034,6 +9103,22 @@ public void testPostRelatedObjectEntryInDifferentCompany()
 		);
 	}
 
+	@Test
+	public void testPostScopeScopeKeyPageWithSiteExternalReferenceCode()
```

Comment:
Could you let me know where you got the name of this method?
The method I saw the name is `postScopeScopeKey` so the name should be `testPostScopeScopeKey`

Can you add the three scenarios (ERC, ID, and name)?